views/settings.php

```diff
@@ -24,25 +46,40 @@
 
 	<?php else : ?>
 
-		<form method="post" action="options.php">
-			<?php settings_fields( 'wp101' ); ?>
+		<div id="wp101-settings-api-key-form" <?php echo ( $api_key ) ? 'class="hide-if-js"' : ''; ?>>
+			<form method="post" action="options.php">
+				<?php settings_fields( 'wp101' ); ?>
 
-			<section id="api-key">
-				<h2><?php echo esc_html( _x( 'WP101Plugin.com API Key', 'settings section heading', 'wp101' ) ); ?></h2>
-				<p><?php esc_html_e( 'Your API key enables your WordPress site to connect to WP101 and retrieve all of your videos.', 'wp101' ); ?></p>
-				<p><strong><?php esc_html_e( 'Don\'t have an API key?', 'wp101' ); ?></strong></p>
-				<p><a href="https://wp101plugin.com" target="_blank"><?php esc_html_e( 'Sign up to get your key now!', 'wp101' ); ?></a></p>
 				<table class="form-table">
 					<th scope="row">
 						<label for="wp101-api-key"><?php esc_html_e( 'API key', 'wp101' ); ?></label>
 					</th>
 					<td>
-						<input name="wp101_api_key" id="wp101-api-key" type="text" value="<?php echo esc_attr( TemplateTags\get_api_key() ); ?>" class="regular-text code" />
+						<input name="wp101_api_key" id="wp101-api-key" type="text" class="regular-text code" />
 					</td>
 				</table>
 
-			</section>
-			<?php submit_button(); ?>
-		</form>
+				<?php submit_button(); ?>
+			</form>
+		</div>
+
+	<?php endif; ?>
+
+	<?php if ( $api_key ) : ?>
```

Comment:
See previous comment - should do a proper check instead of casting to bool due to `$api_key` having the possibility of being null.